Fixes error

In file included from dynamic-link.h:21,
                 from dl-load.c:32:
../sysdeps/mips/dl-machine.h:454: error: parse error before "$29"
../sysdeps/mips/dl-machine.h:454: warning: type defaults to `int' in declaration of `$29'
../sysdeps/mips/dl-machine.h:454: error: stray '\' in program
../sysdeps/mips/dl-machine.h:454: error: syntax error at '#' token
../sysdeps/mips/dl-machine.h:454: error: stray '\' in program
../sysdeps/mips/dl-machine.h:454: error: stray '\' in program
../sysdeps/mips/dl-machine.h:454: error: syntax error at '#' token
../sysdeps/mips/dl-machine.h:454: error: stray '\' in program
../sysdeps/mips/dl-machine.h:454: error: stray '\' in program
../sysdeps/mips/dl-machine.h:454: error: stray '\' in program
../sysdeps/mips/dl-machine.h:454: error: stray '\' in program
../sysdeps/mips/dl-machine.h:454: error: missing terminating " character
...
make[2]: *** [/home/dank/wk/crosstool-0.28-rc24/build/mipsel-unknown-linux-gnu/gcc-3.4.0-glibc-2.2.5/build-glibc/elf/dl-load.o] Error 1
make[2]: Leaving directory `/home/dank/wk/crosstool-0.28-rc24/build/mipsel-unknown-linux-gnu/gcc-3.4.0-glibc-2.2.5/glibc-2.2.5/elf'

See also http://sources.redhat.com/ml/crossgcc/2003-12/msg00014.html

===================================================================
diff -u -r1.58 -r1.59
--- libc/sysdeps/mips/dl-machine.h	2002/02/01 01:31:55	1.58
+++ libc/sysdeps/mips/dl-machine.h	2002/02/03 00:29:15	1.59
@@ -449,7 +449,7 @@
 	addu $7, $7, 4\n\
 	subu $29, 16\n\
 	# Call the function to run the initializers.\n\
-	jal _dl_init
+	jal _dl_init\n\
 	addiu $29, 16\n\
 	# Pass our finalizer function to the user in $2 as per ELF ABI.\n\
 	la $2, _dl_fini\n\
